app/src/main/java/com/sanjeevyadavit/magecart/viewmodel/MainActivityViewModel.kt

```diff
@@ -0,0 +1,9 @@
+package com.sanjeevyadavit.magecart.viewmodel
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class MainActivityViewModel: ViewModel() {
+    val isLoggedIn: MutableLiveData<Boolean> = MutableLiveData(false)
```

Comment:
You must be changing its value from an activity or fragment i guess. 
Expose function in viewModel to modify this data. And use same pattern for this variable mentioned above.